docs/docs/kb/07-frontend.md

```diff
@@ -11,17 +11,17 @@ The frontend app is built using the `@starport/vue` and `@starport/vuex` package
 
 ## Client code generation
 
-JavaScript (JS), TypeScript (TS), and Vuex clients are automatically generated for your blockchain for custom and standard Cosmos SDK modules.
+A TypeScript (TS) client is automatically generated for your blockchain for custom and standard Cosmos SDK modules.
 
 To enable client code generation, add the `client` entries to `config.yml`:
 
 ```yaml
 client:
-  vuex:
+  typescript:
     path: "js"
 ```
 
-A Vuex client is generated in the `js` directory. JS and TS clients are also generated because they are dependencies of the Vuex client.
+A TS client is generated in the `vue/src/generated` directory.
```

Comment:
I believe the path is a different one now, where the parent folder name is `ts-client` instead of `vue`, so the `vue/src/generated` should be replaced by the new path.